stock_reserve_rule/models/stock_reserve_rule.py

```diff
@@ -152,6 +154,32 @@ class StockReserveRuleRemoval(models.Model):
         "When empty, any packaging can be removed.",
     )
 
+    reserve_lot = fields.Selection(
+        selection=[
+            ("lot_qty_eq_requested_qty", "Lot qty = requested qty"),
+            ("lot_qty_gt_requested_qty", "Lot qty > requested qty"),
+            ("lot_qty_lt_requested_qty", "Lot qty < requested qty"),
+        ],
+        string="Requested lot if",
+    )
```

Comment:
Can you elaborate why you need all these?
Finding a single lot should compare quantity with `>=`
Finding an exact lot should compare quantity with '='. Maybe this should be a different strategy.
I don't understand `>` or `<`

Reply:
Hi @jbaudoux , this is a use case where it's important that all quants dispatched to customer are from the same lot (specific textile product processing), so the idea is to use a sequence of different rules to be granular on lot selection priority, eg: 

- first, look for a lot with exact qty as requested
- if not found, look for a lot with qty higher than requested, with a 10% tolerance
- if not found, look for a lot with qty higher than requested, with a 20% tolerance
- if not found, check if there's a lot with less qty than requested, with a 10% tolerance. We will check with customer if a lower qty can work for them. 

This is why we removed >= and =< as there was less control on lot selection.

Reply:
@jbaudoux can we mark this as outdated?